kafka-ui-api/src/main/java/com/provectus/kafka/ui/serde/schemaregistry/SchemaRegistryAwareRecordSerDe.java

```diff
@@ -68,8 +68,9 @@
   private final ObjectMapper objectMapper = new ObjectMapper();
 
   private static SchemaRegistryClient createSchemaRegistryClient(KafkaCluster cluster) {
-    Objects.requireNonNull(cluster.getSchemaRegistry());
-    Objects.requireNonNull(cluster.getSchemaRegistry().getUrl());
+    if (cluster.getSchemaRegistry() == null || cluster.getSchemaRegistry().getUrl() == null) {
```

Comment:
as far as I see there should not be situation when getSchemaRegistry() != null but getSchemaRegistry().getUrl() == null ? If so, getUrl() != null check is redundant